.github/workflows/build.yml

```diff
@@ -20,7 +20,13 @@ jobs:
 
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
+
+      - name: Install libudev-dev on Ubuntu
+        if: matrix.os == 'ubuntu-latest'
```

Comment:
This needs to be `buildjet-8vcpu-ubuntu-2004` because I upgraded to using BuildJet for Ubuntu runners.

Reply:
Done.